packages/frontend/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "albion-mapper",
+  "version": "0.1.0",
+  "private": true,
+  "proxy": "http://localhost:8080",
+  "license": "GNU AGPLv3",
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "lint": "eslint --ext .js,.jsx,.ts,.tsx --ignore-pattern '/node_modules/' ./src",
+    "lint:fix": "eslint --fix --ext .js,.jsx,.ts,.tsx --ignore-pattern '/node_modules/' ./src"
+  },
+  "dependencies": {
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.56",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "@types/cytoscape": "^3.14.7",
```

Comment:
Would be good to have `@types` as devDependencies

Reply:
I'm always conflicted where to put them.